data/card_catalogue/weapons.json

```diff
@@ -18,10 +18,10 @@
     "tier": "green",
     "details": {
       "rules": {
-        "active": "Deal 10 damage to a visible target.",
+        "active": "Cut a 1 meter gash onto a visible surface",
         "passive": "Roll an additional d6 when you perform a melee attack."
       },
-      "flavour": "It didn't slash or pierce, but rather burned. The fallen glowed as though their souls were leaving their bodies.",
+      "flavour": "Master swordsmen can resheath their blade before their opponent knows they've been cut.",
```

Comment:
I felt that 10 damage was a boring effect for such a cool looking card.

Might be too strong, I'm not sure. The visible surface clause at least lets characters override this by wearing clothing. You could also say that this cut is just skin deep, since that's the surface you see.

Reply:
Not overly sold on the active here, but I do think that we need something that plays into the swordsman fantasy.